log/get_logger.ts

```diff
@@ -73,7 +74,7 @@ export function getLogger(name?: string): Logger {
   }
   const result = state.loggers.get(name);
   if (!result) {
-    const logger = new Logger(name, "NOTSET", { handlers: [] });
+    const logger = new Logger(name, DEFAULT_LEVEL, { handlers: [] });
```

Comment:
I wonder if throwing here would make more sense. It might be confusing to return default-leveled logger for invalid logger name

Reply:
Nevermind. I now don't think throwing makes sense after reading the `For module authors` section of the module document ( https://jsr.io/@std/log ). The module authors seem supposed to call `getLogger` with non-defined logger name.